packages/examples/cvat/recording-oracle/src/cvat/interface.py

```diff
@@ -0,0 +1,14 @@
+from pydantic import BaseModel
+
+
+class _AnnotationInfo(BaseModel):
+    accuracy: float | int
+
+
+class _FrameResult(BaseModel):
+    conflicts: list[dict]
+    annotations: _AnnotationInfo
```

Comment:
If `QualityReportData` is public, it's strange to make it's public components private.